spec/player_spec.rb

```diff
@@ -0,0 +1,45 @@
+require './lib/player'
+require 'rspec'
+
+describe Player do
+  context '#initialize' do
+    it 'exists' do
+      board = double('board')
```

Comment:
Oh `double` is new to me. Would like to learn about this.

Reply:
mocking a board